src/LangStrings.inc.h

```diff
@@ -300,6 +300,8 @@ DECLARE_STRING(N_EARTH_RADII)
 DECLARE_STRING(N_SOLAR_RADII)
 DECLARE_STRING(SURFACE_TEMPERATURE)
 DECLARE_STRING(SURFACE_GRAVITY)
+DECLARE_STRING(ESCAPE_VELOCITY)
+DECLARE_STRING(MEAN_DENSITY)
```

Comment:
By the by, you only need to `DECLARE_STRING` here if you're using the string from C++ code as `Lang::ESCAPE_VELOCITY`. Lua has a separate mechanism to load translation strings. I appreciate it being declared for completeness' sake, but I'd like to move description generation entirely into Lua anyways.

Reply:
OK, I think I get it. I grab a random string from that code part, `ORBITAL_PERIOD`, and it seam like it also should not be declared in `src/LangStrings.inc.h`. I can drop the changes I did to this file as they seem pointless. I find deleting things oddly satisfying. :wink:

Reply:
At this juncture, I'd like to leave those strings in. If interested in deleting things, you could port `SystemBody::GetAstroDescription()` over to lua and delete the C++ version and the LangStrings defines... :wink:

To get you started, the lang strings for all of the stars could be handled by this very simple code, as the string versions of the enum so happen to be exactly the same as the translation keys:
```lua
if sbody.superType == "STAR" then
    return lcore[sbody.type]
else
    -- ...
end
```